src/locales/de/ErrorPage.json

```diff
@@ -1,4 +1,7 @@
 {
-    "Subtitle": "Seite nicht gefunden",
-    "MainText": "Diese Seite konnte nicht gefunden werden. Vielleicht besteht ein Fehler in der eingegebenen URL oder auf unserer Serverseite. Bitte entschuldigen Sie die entstandenen Unannehmlichkeiten und kontaktieren Sie gern unser Team unter dem Reiter 'Probleme melden'."
+    "PageNotFoundTitle": "Seite nicht gefunden",
+    "PageNotFoundText": "Diese Seite konnte nicht gefunden werden. Vielleicht besteht ein Fehler in der eingegebenen URL oder auf unserer Serverseite. Gern können Sie  unser Team unter dem Reiter 'Probleme melden' konatktieren.",
+    "GeneralErrorTitle": "Ein Fehler ist aufgetreten.",
+    "GeneralErrorText": "Vielleicht besteht ein Fehler auf unserer Serverseite. Gern können Sie  unser Team unter dem Reiter 'Probleme melden' konatktieren."

```

Comment:
typo in both messages: konatkieren.
But you said that the messages are only a placeholder, so no reject